Makefile

```diff
@@ -48,6 +48,8 @@ dist-zip: clean
 	mv .$(DIST) $(DIST)
 	$(MAKE) -C $(DIST) install-vendor # install the composer vendor inside the dist dir
 	rm $(DIST)/Makefile # this step needs to be the last step before zipping
+	rm $(DIST)/release.sh
+	cp .version $(DIST)/.version
```

Comment:
Could we move this line before line 48 above, so by the time we rename the directory, all the expected content is present